kitchen-tests/cookbooks/end_to_end/recipes/_homebrew_tap.rb

```diff
@@ -0,0 +1,5 @@
+homebrew_tap 'chef/chef'
```

Comment:
You'll need to include this in the main test recipe for macos

Reply:
@tas50 this should be good to go. I added the _homebrew_tap test to the macOS test recipe and it is passing.